azurerm/config.go

```diff
@@ -492,6 +496,7 @@ func getArmClient(c *authentication.Config) (*ArmClient, error) {
 	client.registerRelayClients(endpoint, c.SubscriptionID, auth, sender)
 	client.registerResourcesClients(endpoint, c.SubscriptionID, auth)
 	client.registerSearchClients(endpoint, c.SubscriptionID, auth)
+	client.registerSecurityCenterClients(endpoint, c.SubscriptionID, "Global", auth)
```

Comment:
this refers to the Azure Location/Reggion `Global` (which imo shouldn't be exposed here in the Swagger/SDK, since it's hard-coded to `Global` but 🤷‍♂️) - but I don't think this is worth a constant imo?

Reply:
Then let's keep the literal here.